projects/gdk-pixbuf/project.yaml

```diff
@@ -0,0 +1,6 @@
+homepage: https://gitlab.gnome.org/GNOME/gdk-pixbuf/
+language: c
+primary_contact: jvoisin@google.com
```

Comment:
Before we merge this, try getting maintainer email [ask them on those threads asap since your internship ending soon]. Julien's email should be in auto_ccs attribute. Same for the other 3 PRs. @Google-Autofuzz

Reply:
I moved Julien's email to auto_ccs and temporarily listed security-tps as the primary contact. I also asked upstream for an email address again so I'll update it as soon as they provide me with one.

Reply:
https://gitlab.gnome.org/GNOME/gdk-pixbuf/-/merge_requests/84#note_971662

I sent them a message yesterday but still no word.

Reply:
I received an email address this evening. I updated my development [branch](https://github.com/ecalp-tps/oss-fuzz/commit/a92d26ad6b7560a0510363f5d99c35e926a93fd6#diff-13be7c9ab3b8b5460c1a8668e11ac3323ff1d091d83570e0b57fa020308b953c) with the address. Since I was still working on Gdk-Pixbuf and Cairo when their PRs were merged, it might be a good idea to revert them and send PRs from their development branches to get the most up-to-date version.

Reply:
This looks like a simple diff, why not just submit a new PR with adding the email address